src/components/QRShare/index.tsx

```diff
@@ -60,7 +60,7 @@ function QRShare({innerRef, url, title, subtitle, logo, logoRatio, logoMarginRat
                 {title}
             </Text>
 
-            {!_.isEmpty(subtitle) && (
+            {subtitle && (
```

Comment:
In case `subtitle` is an empty string it can trigger a console warning on the web platforms
```suggestion
            {!!subtitle && (
```

Reply:
Nice catch @VickyStash ! Tested here and the error happened, will commit a fix.